test/scriptFlywheel/unit/liquidateTests.spec.ts

```diff
@@ -952,6 +928,55 @@ contract("SavingAccount.liquidate", async (accounts) => {
                     await mockChainlinkAggregatorforDAI.updateAnswer(DAIprice);
                     await mockChainlinkAggregatorforDAI.updateAnswer(DAIprice);
                 });
+
+                it("when borrow LTV is greater than 95% (for ETH)", async function () {
+                    this.timeout(0);
+                    const borrowAmt = new BN(await tokenInfoRegistry.priceFromIndex(0))
+                        .mul(new BN(60))
+                        .div(new BN(100))
+                        .mul(ONE_ETH)
+                        .div(new BN(await tokenInfoRegistry.priceFromIndex(9)));
+                    await erc20DAI.transfer(user1, ONE_DAI);
+                    await erc20DAI.approve(savingAccount.address, ONE_DAI, { from: user1 });
+                    await savingAccount.deposit(addressDAI, ONE_DAI, { from: user1 });
+                    await savingAccount.deposit(ETH_ADDRESS, ONE_ETH, {
+                        value: ONE_ETH,
+                    });
+                    await savingAccount.deposit(ETH_ADDRESS, ONE_ETH, {
+                        from: user2,
+                        value: ONE_ETH,
+                    });
+                    // 2. Start borrowing.
+                    const result = await tokenInfoRegistry.getTokenInfoFromAddress(addressDAI);
+                    const daiTokenIndex = result[0];
+                    await accountsContract.methods["setCollateral(uint8,bool)"](
+                        daiTokenIndex,
+                        true,
+                        { from: user1 }
+                    );
+                    await savingAccount.borrow(ETH_ADDRESS, borrowAmt, { from: user1 });
+                    // 3. Change the price.
+                    let updatedPrice = new BN(1);
+                    let originPrice = await mockChainlinkAggregatorforDAI.latestAnswer();
+
+                    await mockChainlinkAggregatorforDAI.updateAnswer(updatedPrice);
+
+                    // Check that borrow LTV > 95%
+                    const userBorrowVal = await accountsContract.getBorrowETH(user1);
+                    const getDeposits = await accountsContract.getDepositETH(user1);
+                    const userBorrowPower = await accountsContract.getBorrowPower(user1);
+                    let UB = new BN(userBorrowVal).mul(new BN(100));
+                    let UD = new BN(getDeposits).mul(new BN(95));
+                    console.log("userBorrowVal", userBorrowVal.toString());
+                    console.log("userBorrowPower", userBorrowPower.toString());
+                    console.log("UD", UD.toString());
+                    console.log("UB", UB.toString());
+
+                    expect(UB).to.be.bignumber.greaterThan(UD);
+
+                    await savingAccount.liquidate(user1, ETH_ADDRESS, addressDAI);
```

Comment:
After liquidation the following should happen:
- The liquidator should get the `collateralTokens` 
- The liquidator's token deposit should reduce.
- The borrowers collateralToken should reduce.
Pls see if you can validate the exact number of the above values.

Reply:
Added checks for these